optimum/intel/neural_compressor/quantization.py

```diff
@@ -535,7 +536,11 @@ def from_pretrained(
             raise ValueError("INC IPEX is currently not supported")
 
         # Load the state dictionary of the model to verify whether the model is quantized or not
-        state_dict = torch.load(state_dict_path)
+        if not cpu_flag: 
+            state_dict = torch.load(state_dict_path)
+        elif cpu_flag:
+            state_dict = torch.load(state_dict_path,map_location='cpu')
```

Comment:
I think this would makes sense then :
```suggestion
        state_dict = torch.load(state_dict_path, map_location="cpu")

```